app/src/containers/CoverPage.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+import CoverPage from '../components/Home/CoverPage';
+import { getCoverPageEntries } from '../actions/coverPage';
+
+const mapStateToProps = (state) => ({
+  coverPageEntries: state.coverPageEntries
+});
```

Comment:
Is the state need it? I don't think we save the status of the coverpage at any moment.